src/components/charts/galaxySelector/index.jsx

```diff
@@ -317,6 +318,24 @@ class GalaxySelector extends React.PureComponent {
     }));
   };
 
+  getAlertDaysAndHours = id => {
+    const { activeGalaxy } = this.props;
+    const { alerts } = activeGalaxy || {};
+    let days = 0;
+    let hours = 0;
+
+    if (alerts) {
+      alerts.forEach(alert => {
+        if (alert.image_id === id) {
```

Comment:
I believe the `getAlertFromImageId` utility method retrieves the `alert` from the `image_id` and the `alerts`.  If so, use that instead of this loop.